.husky/commit-msg

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx --no -- commitlint --edit "$1"
```

Comment:
Is there supposed to be a space there?

```suggestion
npx --no --commitlint --edit "$1"
```

Reply:
ah yeah the tool auto generates this file so that is really strange. I am tempted to say leave it as its generated